MDCDecorator/MDCDecoratorEnd.yml

```diff
@@ -0,0 +1,11 @@
+steps:
+- task: Bash@3
+  displayName: MDC Extension End
+  timeoutInMinutes: 1
+  condition: always()
+  continueOnError: true
+  inputs:
+    targetType: 'inline'
+    script: | 
+      echo "Pipeline Start: $(pipelineStartTime)"
+      docker images --filter "since=$(firstDocker)" --format "table {{.CreatedAt}} {{.Repository}} {{.Digest}}" 
```

Comment:
We don't need the ` || true` at the end of the scripts because of the `continueOnError: true` parameter provided by ADO, this ensures that even on failure this task wont block pipeline. more info [here](https://learn.microsoft.com/en-us/azure/devops/pipelines/process/tasks?view=azure-devops&tabs=yaml#task-control-options)